src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -252,6 +252,39 @@ const PersonalInfoList = ({
       />
     ) : null;
 
+  const note = isFacStaff ? (
+    <Typography align="left" className="note">
+      NOTE:
+      <ul>
+        <li>
+          To prevent your picture from displaying click{' '}
+          <a href="https://360.gordon.edu/myprofile"> here</a>.
+        </li>
+        <li>
+          To update your data contact <a href="mailto: hr@gordon.edu">Human Resources</a> (x4828).
+        </li>
+      </ul>
+    </Typography>
+  ) : isStudent ? (
+    <Typography align="left" className="note">
+      NOTE:
+      <ul>
+        <li>
+          To prevent your picture or your cell phone number from displaying, click{' '}
+          <a href="https://360.gordon.edu/myprofile">here</a>.
+        </li>
+        <li>
+          To update your On Campus Address, contact <a href="mailto: housing@gordon.edu">housing</a>
```

Comment:
Could fix the capitalization on "housing" to be "Housing" as it is a department name, and it mirrors the "Registrar's Office" on line 282 for consistency.

Reply:
Good Catch!! Thanks!